test/java/seedu/addressbook/util/TypicalPersons.java

```diff
@@ -34,6 +37,11 @@ public TypicalPersons() {
         }
     }
 
+    /**
+     * Inserts predefined {@code Person} objects in to a given instance of
+     * {@code AddressBook}
```

Comment:
Can be in one line?

Reply:
Side thought that doesn't need to be acted on at all: Isn't it more important that we document the *public* methods `getTypicalPersons()` and `getTypicalAddressBook()` rather than documenting this? :thinking:

Reply:
I would honestly rather not be writing JavaDocs for simple, short functions...